src/Components/SavedSearchAlert/Components/FrequencyRadioButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import { BorderedRadio, Box, RadioGroup, Text } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import styled from "styled-components"
+import { useSystemContext } from "System"
+import { isArtsyEmail } from "Utils/isArtsyEmail"
+import { SavedSearchFrequency } from "../types"
+
+interface FrequenceRadioButtonsProps {
+  defaultFrequence: SavedSearchFrequency
+  onSelect: (selectedOption: string) => void
+}
+
+export const FrequenceRadioButtons: React.FC<FrequenceRadioButtonsProps> = ({
+  defaultFrequence,
+  onSelect,
+}) => {
+  const { user } = useSystemContext()
+  const isArtsyEmployee = isArtsyEmail(user?.email ?? "")
+
+  if (isArtsyEmployee) {
+    return (
+      <Box>
+        <Text variant="sm-display" mb={1}>
+          Frequency
+        </Text>
+        <RadioGroup defaultValue={defaultFrequence} onSelect={onSelect}>
+          <BorderedRadio
+            value="instant"
+            label={
+              <Text>
+                Instant <Warning>(experimental feature)</Warning>
```

Comment:
**suggestion**: Thoughts in including admin only in this message?

Instant (beta / admin only)

Reply:
updated. you can check changes in review app